packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -961,6 +961,19 @@ describe('plugin-meetings', () => {
           await meeting.addMedia().catch((err) => {
             assert.exists(err);
             assert.isNull(meeting.statsAnalyzer);
+            assert(Metrics.sendBehavioralMetric.calledOnce);
```

Comment:
it would be good to have at least 1 test for the case where turnDiscoverySkippedReason is not undefined 

BTW, these existing tests are not very good, because they will pass if addMedia() succeeds and doesn't send any metrics at all as the catch() code will not be executed at all in such case, but don't try to fix that now as I've refactored them on the sdk_v3 branch.